core/domain/feedback_jobs_continuous.py

```diff
@@ -93,6 +93,7 @@ def _handle_incoming_event(cls, active_realtime_layer, event_type, *args):
         """
         exp_id = args[0]
 
+        @transaction_services.run_in_transaction_wrapper
         def _increment_open_threads_count():
```

Comment:
Maybe add transactional in the name to make sure at the callsites it is explicit.

Reply:
Done!